src/Stryker.CLI/Stryker.CLI/Logging/ApplicationLogging.cs

```diff
@@ -49,5 +36,30 @@ public static ILoggerFactory LoggerFactory
             get => _factory ??= new LoggerFactory();
             set => _factory = value;
         }
+
+        private static class LogLevelConverter
+        {
+            public static LibGitLogLevel Convert(LogEventLevel level) => level switch
+            {
+                LogEventLevel.Information => LibGitLogLevel.None,
```

Comment:
Is it intentional that `Information` does not map to `Info`?

Reply:
Yes. Libgit2sharp logging is not really necessary normally and can be quite spammy so we don't want it enabled in normal scenarios.